universal/ubi8/Dockerfile

```diff
@@ -453,8 +451,9 @@ RUN echo '[[ -s "/home/user/.sdkman/bin/sdkman-init.sh" ]] && source "/home/user
 # Set permissions on /etc/passwd and /home to allow arbitrary users to write
 RUN mkdir -p /home/user && chgrp -R 0 /home && chmod -R g=u /etc/passwd /etc/group /home
 
-# cleanup dnf cache
-RUN dnf -y clean all --enablerepo='*'
+# cleanup dnf & yum cache
+RUN dnf -y clean all --enablerepo='*' \
+    && rm -rf /var/cache/yum
```

Comment:
Cleaning everything as a separate step doesn't have the effect we would want it to (it actually increases image size unless the image is squashed)

Reply:
Thanks for the tip @amisevsk, will revert the change 👍